spec/features/staff/users_can_edit_an_activity_spec.rb

```diff
@@ -222,10 +222,18 @@ def assert_all_edit_links_go_to_the_correct_form_step(activity:)
   end
   click_on(I18n.t("generic.link.back"))
 
+  within(".geography") do
+    click_on(I18n.t("generic.link.edit"))
+    expect(page).to have_current_path(
+      activity_step_path(activity, :geography)
+    )
+  end
+  click_on(I18n.t("generic.link.back"))
+
   within(".recipient_region") do
     click_on(I18n.t("generic.link.edit"))
     expect(page).to have_current_path(
-      activity_step_path(activity, :country)
+      activity_step_path(activity, :region)
```

Comment:
Shouldn't we have something like this for `country`, too?

Reply:
Same as previous, I feel like that is here:

https://github.com/UKGovernmentBEIS/beis-report-official-development-assistance/blob/feature/303-activities-collect-recipient-country/spec/features/staff/users_can_manage_activity_geography_spec.rb

We have the most feature specs I have ever worked with and I've found it hard to navigate them, I feel we could end up with giant features that try to do too much and are (even more) difficult to follow.